apps/dashboard/src/hooks/useTxNotifications.ts

```diff
@@ -15,18 +7,7 @@ import { useInvalidateContractQuery } from "thirdweb/react";
 export function useTxNotifications(
   successMessage: string,
   errorMessage: string,
-  contract?: // v4 types
-    | SmartContract
-    | NFTContract
-    | TokenContract
-    | Erc1155
-    | Erc721
-    | Erc20
-    | ValidContractInstance
-    | undefined
-    // v5...
-    | ThirdwebContract
-    | null,
+  contract?: ThirdwebContract | null,
 ) {
   const toast = useToast();
```

Comment:
while you're at it could you also replace this toast with the `toast` from `sonner` library? :)

Reply:
sure thing